src/util/RouterUtil.js

```diff
@@ -131,6 +143,14 @@ function (Okta, Util, OAuth2Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
         successData.session = {
           token: res.sessionToken,
           setCookieAndRedirect: function (redirectUrl) {
+            if (suppliedExtension) {
+              var loginPageToOkta = {
+                'gsd-sendgrid.cs18.force.com': 'https://ctlpreview.oktapreview.com',
+                'home.crisistextline.org': 'https://crisistextline.okta.com',
```

Comment:
I feel like these values should be provided from the environment that the JS file is hosted in... especially in an open-source fork of a public repo (like this one). That would require getting some values from SF in the Visualforce page and passing them into the function whenever we call it. @madned what do you think?

Reply:
These aren't hard coded, and are determined at build time. The requirements.txt is the file that determines which versions. When dealing with python libraries, you don't determine based on the system. It's a portable language that way, with specific deps.